lua/autorun/client/cl_apple.lua

```diff
@@ -6,6 +6,7 @@ end)
 
 net.Receive( "cfc_playerinitialspawn", function( len )
```

Comment:
Why does this take an entity? Couldn't we just send the info about the player over to the client? 

Hell, we could even send the entire string to print to the client's chat box.

Can we do this from the serverside actually? Or can we not color the text with `ply:ChatPrint` ?

Reply:
It sends the entity so I don't have to send everything else like the players name, their team, stuff like that. And no, you can color with `ply:ChatPrint` .

Reply:
Well, if we got the info on the serverside then I think we'd have a better chance of actually getting the player's info. Just seems like we shouldn't send an entire entity to the client when we only need a few bits of text